Formula/stroke.rb

```diff
@@ -0,0 +1,17 @@
+class Stroke < Formula
+  desc "🖊️ A MacOS command stroke tool written in Rust 一款基于 Rust 实现的 Mac 命令行一笔画绘线工具"
+  homepage "https://github.com/meloalright/stroke"
+  url "https://github.com/meloalright/stroke/archive/refs/tags/0.1.5.tar.gz"
+  sha256 "cd20d8647272dcedba5eb779e29a7eebe8ec21ec9e2947db893969052bc1aa7a"
+  license "MIT"
+
+  depends_on "rust" => :build
```

Comment:
Given the description:
```suggestion
  depends_on "rust" => :build
  depends_on :macos
```

Reply:
thanks a lot

I processed it in [4ed513e](
https://github.com/meloalright/homebrew-core/blob/4ed513e1e6e0e9636bdbc127181537fa31f40518/Formula/stroke.rb#L8-L9)